acceptance/openstack/ces/v1/events_test.go

```diff
@@ -0,0 +1,28 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ces/v1/events"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestEvents(t *testing.T) {
+	client, err := clients.NewCesV1Client()
+	th.AssertNoErr(t, err)
+
+	eventsRes, err := events.ListEvents(client, events.ListEventsOpts{
```

Comment:
Fails with: Failure in events_test.go, line 18: unexpected error "Resource not found: [GET https://ces.eu-ch2.sc.otc.t-systems.com/V1.0/b4372bf357a54745b311a8bca742feed/event], error message: {\"error_msg\":\"The API does not exist or has not been published in the environment\",\"error_code\":\"APIGW.0101\",\"request_id\":\"63373c12d550d1661dc069c496f34e52\"}\n"